### PR TITLE
Added find_account_by_id to crossbar_maintenance

### PR DESCRIPTION
**Pull request by danielfinke**

### CODE REVIEW

**Comment by danielfinke** [comment]
Updated this PR to remove dependency on #1257

**Comment by lazedo** [comment]
thanks, will merge it when travis agrees.

File: applications/crossbar/src/crossbar_maintenance.erl [hunk 1]
```diff
@@ -271,6 +272,20 @@ find_account_by_realm(Realm) ->
 %%
 %% @end
 %%--------------------------------------------------------------------
+-spec find_account_by_id(input_term()) ->
+                                   {'ok', ne_binary()} |
+                                   {'error', term()}.
+find_account_by_id(Id) when not is_binary(Id) ->
+    find_account_by_id(wh_util:to_binary(Id));
+find_account_by_id(Id) ->
+    print_account_info(wh_util:format_account_id(Id, 'encoded')).
```
Comment (fenollp) [inline applications/crossbar/src/crossbar_maintenance.erl]: I would do the opposite and have a guard like `when is_binary…` then an unguarded clause

**Comment by lazedo** [comment]
@danielfinke in the future, please make pull request against master and ask to backport if needed

**Pull request merged**